### PR TITLE
Update runtime Brooklyn

### PR DESCRIPTION
**Pull request by obsessed-cake**

### CODE REVIEW

File: runtime/brooklyn/src/ethereum.rs [hunk 1]
```diff
@@ -29,122 +22,13 @@ pub const GAS_PER_SECOND: u64 = 40_000_000;
...
-}
-
 parameter_types! {
 	pub BlockGasLimit: U256 = U256::from(
 		NORMAL_DISPATCH_RATIO * BlockWeights::get().max_block.ref_time() / WEIGHT_PER_GAS
 	);
 	pub PrecompilesValue: GoldenGatePrecompiles<Runtime> = GoldenGatePrecompiles::<_>::new();
 	pub WeightPerGas: Weight = Weight::from_parts(WEIGHT_PER_GAS, 0);
-	pub ChainId: u64 = 888866;
+	pub ChainId: u64 = 8886;
```
Comment (akorchyn) [inline runtime/brooklyn/src/ethereum.rs:31]: chain id for brooklyn is not decided, but 8886 is for sydney

File: runtime/brooklyn/src/pos/currency.rs [hunk 2]
```diff
@@ -0,0 +1,965 @@
+// TODO: benchmark and set proper weight for calls
+
+use core::marker::PhantomData;
```
Comment (akorchyn) [inline runtime/brooklyn/src/pos/currency.rs:3]: It's a pallet. Let's extract it into common

File: runtime/brooklyn/src/pos/session_payout/mod.rs [hunk 3]
```diff
@@ -0,0 +1,457 @@
+// TODO: benchmark and set proper weight for calls
```
Comment (akorchyn) [inline runtime/brooklyn/src/pos/session_payout/mod.rs:1]: same

File: runtime/brooklyn/src/ethereum.rs (hunk 1 above)
Comment (obsessed-cake) [reply runtime/brooklyn/src/ethereum.rs]: good catch.
fixed.

File: runtime/brooklyn/src/pos/currency.rs (hunk 2 above)
Comment (obsessed-cake) [reply runtime/brooklyn/src/pos/currency.rs]: Fixed.

File: runtime/brooklyn/src/pos/session_payout/mod.rs (hunk 3 above)
Comment (obsessed-cake) [reply runtime/brooklyn/src/pos/session_payout/mod.rs]: Done.

**Review by akorchyn — APPROVED** [review]

**Pull request merged**